src/OrbitQt/CaptureOptionsDialog.cpp

```diff
@@ -194,6 +225,40 @@ DynamicInstrumentationMethod CaptureOptionsDialog::GetDynamicInstrumentationMeth
   ORBIT_UNREACHABLE();
 }
 
+void CaptureOptionsDialog::SetEnableCallStackCollectionOnThreadStateChanges(bool check) {
+  ui_->threadStateChangeCallstackCollectionCheckBox->setChecked(check);
+}
+
+bool CaptureOptionsDialog::GetEnableCallStackCollectionOnThreadStateChanges() const {
+  return ui_->threadStateChangeCallstackCollectionCheckBox->isChecked();
+}
+
+void CaptureOptionsDialog::SetThreadStateChangeCallstackMethod(
+    CaptureOptions::UnwindingMethod thread_state_change_callstack_method) {
+  switch (thread_state_change_callstack_method) {
+    case CaptureOptions::kFramePointers:
+      ui_->threadStateChangeCallstackFramepointersMethodRadioButton->setChecked(true);
+      ui_->threadStateChangeCallstackDWARFMethodRadioButton->setChecked(false);
+      break;
+    case CaptureOptions::kDwarf:
+    default:
```

Comment:
Why the default here? I will move it below and log some error.

Reply:
Did I do it correctly now?